extensions/amp-story/1.0/amp-story.js

```diff
@@ -2471,28 +2446,99 @@ export class AmpStory extends AMP.BaseElement {
     }
   }
 
+  /**
+   * Adds the localization string bundles to the localization service.
+   */
+  installLocalizationStrings_() {
+    const localizationService = getLocalizationService(this.element);
+    const storyLanguage = localizationService.getLanguageCodesForElement(
+      this.element
+    )[0];
+    if (this.registerInlineLocalizationStrings_(storyLanguage)) {
+      return;
+    }
+    if (isExperimentOn(this.win, 'story-remote-locales')) {
```

Comment:
Plz report the experiment as enabled to the CSI metrics so we can filter our dashboards